docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Scan your first project with Semgrep
+hide_title: false
+description: Learn how to set up Semgrep and scan your first project
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.7 or later installed on the machine where the Semgrep CI is running.
+:::
+
+1. Navigate to [Semgrep](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
```

Comment:
```suggestion
1. Navigate to [Semgrep](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
```
Do you remember which of those permissions they asked for? We have a list of permissions, but I believe they are required at different stages of the user journey now.

Reply:
Can we add a section somewhere (maybe at the bottom) for users who may want to know this prior to signing up? You can reuse from this: [https://semgrep.dev/docs/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab](https://semgrep.dev/docs/semgrep-cloud-platform/getting-started/#requested-permissions-for-github-and-gitlab)